src/main/lens-proxy-port-number-state.injectable.ts

```diff
@@ -0,0 +1,36 @@
+/**
```

Comment:
nit: I think having all the code concerning "lens-proxy" should be in their own folder.

Reply:
Agree, done.